src/wp-includes/sitemaps.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Sitemaps: Public functions
+ *
+ * This file contains a variety of public functions developers can use to interact with
+ * the XML Sitemaps API.
+ *
+ * @package WordPress
+ * @subpackage Sitemaps
+ * @since 5.5.0
+ */
+
+/**
+ * Retrieves the current Sitemaps server instance.
+ *
+ * @since 5.5.0
+ *
+ * @return WP_Sitemaps|null Sitemaps instance, or null if sitemaps are disabled.
+ */
+function wp_sitemaps_get_server() {
+	/**
+	 * Global Core Sitemaps instance.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @var WP_Sitemaps $wp_sitemaps
+	 */
+	global $wp_sitemaps;
+
+	$is_enabled = (bool) get_option( 'blog_public' );
+
+	/**
+	 * Filters whether XML Sitemaps are enabled or not.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param bool $is_enabled Whether XML Sitemaps are enabled or not. Defaults to true for public sites.
+	 */
+	$is_enabled = (bool) apply_filters( 'wp_sitemaps_is_enabled', $is_enabled );
```

Comment:
Make a new function from this, `wp_is_sitemaps_enabled()`?

Reply:
[swissspidy, via Slack](https://wordpress.slack.com/archives/CTKTGNJJW/p1591712892444900):

`wp_sitemaps_get_server` returns `null`  if sitemaps are not enabled.
And if they are enabled, sitemaps will have been initialzied by core, so you’ll just get the sitemaps instance